Add ORDER field to Sort Command (#108)
AY2425S1-CS2103T-F14A-3/tp: docs/UserGuide.md

Sorts the client list in ascending order by the specified field.

> Format: sort `FIELD ORDER`

Reviewer: ```suggestion
Format: `sort FIELD ORDER`
```

> * `FIELD` can be either `deadline` or `name`
> * `ORDER` can be either `ascending` or `descending`

Reviewer: ```suggestion
* `FIELD` can be either `deadline` or `name` (**case-insensitive**)
* `ORDER` can be either `ascending` or `descending` (**case-insensitive**)
```

Examples:
* `sort name ascending` sorts the client list alphabetically by name in ascending order.
* `sort deadline descending` sorts the client list by project deadline (later dates first).

<div markdown="span" class="alert alert-primary">:bulb: **Tip:**

> Addressbook saves you the hassle of sorting the list every time you open the app, so everytime you do a sort, it will save the addressbook

Reviewer: ```suggestion
Clientele+ saves you the hassle of sorting the client list every time you open the app, so every time you do a sort, it will save the list.
```
